content/articles/2018-05-pkgdown-1-0-0.Rmd

```diff
@@ -0,0 +1,68 @@
+---
+title: pkgdown 1.0.0
+author: Hadley Wickham
+date: '2018-05-03'
+slug: pkgdown-1-0-0
+categories: [packages]
+photo:
+  url: https://unsplash.com/photos/rx1iJ59jRyU
+  author: Annie Spratt
+---
+
+I'm very pleased to announce the initial release of [pkgdown](http://pkgdown.r-lib.org) to CRAN! pkgdown is designed to make it quick and easy to build a website for your package. You can see pkgdown in action at <http://pkgdown.r-lib.org>: this is the output of pkgdown applied to the latest version of pkgdown. This initial release is version 1.0.0 in recognition that pkgdown has been in development for 6 years (!) and has already been used to make websites for [over 1600 packages](https://github.com/search?q=pkgdown.css+in%3Apath+path%3Adocs&type=Code) (!!). 
+
+Install pkgdown with:
+
+```{r, eval = FALSE}
+install.packages("pkgdown")
+```
+
+## Usage
+
+Basic usage is simple: run `build_site()` within your package directory:
+
+```{r, eval = FALSE}
+pkgdown::build_site()
+```
+
+By default, this will generate a `docs/` directory. The home page will be generated from your `README.md`, a function reference will be generated from the documentation in the `man/` directory, and any `vignettes/` will be converted into articles
+
+If you are using GitHub, the easiest way to make this your package website is to check it in, push it, then go repo settings ensure that the __GitHub pages__ source is set to "master branch /docs folder".
+
+## Customisation
+
+To customise your site, first read [`vignette("pkgdown")`](http://pkgdown.r-lib.org/articles/pkgdown.html), then read the documentation for the functions responsible for building each part of the site:
+
+* [`init_site()`](init_site.html)
+* [`build_home()`](build_home.html)
+* [`build_reference()`](build_reference.html)
+* [`build_articles()`](build_articles.html)
+* [`build_tutorials()`](build_tutorials.html)
+* [`build_news()`](build_news.html)
+
+## In the wild
+
+A great way to see what you can do with pkgdown is to look at existing website. As well as all the tidyverse websites, here are a few examples created by some of the contributors to pkgdown:
```

Comment:
I might cut the "As well as all the tidyverse websites," and just jump right in with "Here are a few examples…"